dev-infrastructure/templates/global-acr.bicep

```diff
@@ -23,16 +27,55 @@ resource globalMSI 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31'
   name: globalMSIName
 }
 
+//
+//   O C P   A C R
+//
+
 module ocpAcr '../modules/acr/acr.bicep' = {
   name: ocpAcrName
   params: {
     acrName: ocpAcrName
     acrSku: ocpAcrSku
+
     location: location
     zoneRedundant: determineZoneRedundancyForRegion(location, ocpAcrZoneRedundantMode)
   }
 }
 
+module ocpCaching '../modules/acr/cache.bicep' = {
+  name: '${ocpAcrName}-caching'
+  params: {
+    acrName: ocpAcrName
+    location: location
+    quayRepositoriesToCache: [
+      {
+        ruleName: 'openshiftReleaseDev'
+        sourceRepo: 'quay.io/openshift-release-dev/*'
+        targetRepo: 'openshift-release-dev/*'
+        userIdentifier: 'quay-username'
+        passwordIdentifier: 'quay-password'
+        loginServer: 'quay.io'
+      }
+    ]
+    purgeJobs: [
+      {
+        name: 'openshift-release-dev-purge'
+        purgeFilter: 'quay.io/openshift-release-dev/.*:.*'
+        purgeAfter: '${ocpImagesPurgeAfterDays}d'
+        imagesToKeep: 100
```

Comment:
when you have `purgeAfter` and `imagesToKeep`, does that mean:
1. we will only purge images after the `purgeAfter` date?
2. we will purge images after the `purgeAfter` date but also only keep 100 images max
3. something else?

Reply:
3) we will purge images after the purgeAfter date but also only keep 100 images *min*
but for OCP images we will switch to retention policy